extras/docker/fromsource/heketi-start.sh

```diff
@@ -74,4 +85,32 @@ if [[ -d "${BACKUPDB_PATH}" ]]; then
     fi
 fi
 
-/usr/bin/heketi --config=/etc/heketi/heketi.json
+# if NEED_TOPOLOGY_LOAD is not empty, start the service in the background, and
+# load the topology.json from the path in HEKETI_TOPOLOGY_FILE
+if [ -z "${NEED_TOPOLOGY_LOAD}" ]; then
+    # by default just start in the foreground
+    /usr/bin/heketi --config=/etc/heketi/heketi.json
+else
+    # start hketi in the background
+    /usr/bin/heketi --config=/etc/heketi/heketi.json &
+
+    # wait until heketi replies
+    while ! curl http://localhost:8080/hello; do
```

Comment:
Do we want to suppress curl's (standard) output here?

Also, heketi's port is configurable in theory. It's generally not changed much though. Any thoughts about that?

Reply:
I thought about it, but it would be good to have some logging in case this decides to hang.

Is there a good way to find out what the port is configured to? Can heketi-cli pick that from the heketi.json maybe?

Reply:
> I thought about it, but it would be good to have some logging in case this decides to hang.

OK, understood. It's not like this code path will run frequently.

> Is there a good way to find out what the port is configured to? Can heketi-cli pick that from the heketi.json maybe?

That's a great idea! :-)

Reply:
No change here. It would be part of a different Issue/PR. If you want this changed, maybe create an Issue for it and assign it to me :)

Reply:
Certainly, I woudln't expect that to be resolved by this pr.